app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -39,6 +40,7 @@ def self.step_info
         preconditions: ->(idv_session:, user:) do
           self.enabled?
         end,
+        undo_step: ->(idv_session:, user:) { true }, # clear any saved data
```

Comment:
i'm not seeing any code that reads the return value from these callbacks, what does `true` do here?

Reply:
Fixed, thanks. I wasn't sure I could use `{}`, but I tried it locally and that's allowed.